src/main/java/gov/cms/madie/services/MeasureMapper.java

```diff
@@ -211,7 +211,7 @@ default List<Object> populationToDefOrAgg(Population population) {
 
   @Named("scoringUnitToUcum")
   default String scoringUnitToUcum(Object scoringUnit) {
-    if (scoringUnit == null) {
+    if (scoringUnit == null || "".equals(scoringUnit)) {
```

Comment:
let's swap this for StringUtils.isBlank

Reply:
which we can do with the next PR since we're still working in this..

Reply:
It's not string always. if value present, it is an object. if not, it could be null or empty string. this needs to be handled in UI while adding/editing to behave consistency.